site/src/api/queries/groups.ts

```diff
@@ -8,14 +8,15 @@ import {
 } from "api/typesGenerated";
 
 const GROUPS_QUERY_KEY = ["groups"];
+type GroupSortOrder = "asc" | "desc";
 
 const getGroupQueryKey = (groupId: string) => ["group", groupId];
 
 export const groups = (organizationId: string) => {
   return {
     queryKey: GROUPS_QUERY_KEY,
     queryFn: () => API.getGroups(organizationId),
-  };
+  } satisfies UseQueryOptions<Group[]>;
```

Comment:
Why not set this type on the function return type instead of use `satisfies`?

Reply:
In this case, it's more just out of habit. With `satisfies`, the return type is inferred as the exact properties from `UseQueryOptions` that are actually being used. Usually it shows up better in Intellisense

With the explicit return type, the type information gets "shadowed" by the full `UseQueryOptions` type definition, and you just know you have some kind of object that works with `useQuery`